UIKit1/ChoiceViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ChoiceViewController.swift
+//  UIKit1
+//
+//  Created by Ольга on 03.11.2022.
+//
+
+import UIKit
+
+final class ChoiceViewController: UIViewController {
+
+    //MARK: IBOutlet
+    @IBOutlet private var pepperoni: UIImageView!
```

Comment:
```suggestion
    @IBOutlet private var pepperoniImageView: UIImageView!
```
всегда добавляй в название переменной название ui элемента